### ISSUE TITLE
Bug report: aes encryption bug if incoming byte >= 0x80

### ISSUE THREAD

**Issue by ComingNine** [gchq/CyberChef#210]
aes encryption bug if incoming byte >= 0x80

### Summary
aes encryption has bug if incoming byte >= 0x80

### Example
Please have a look at the link below: an incoming byte 0xff is encrypted to two bytes O_O

* Version: 6.5.0
* Compile time: 24/11/2017 16:40:58 UTC
* User-Agent: 
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/62.0.3202.94 Safari/537.36
* [Link to reproduce](https://gchq.github.io/CyberChef/#recipe=From_Hex('Space')AES_Encrypt(%7B'option':'Hex','string':'11223344556677889900aabbccddeeff'%7D,%7B'option':'Hex','string':'11223344556677889900aabbccddeeff'%7D,%7B'option':'Hex','string':''%7D,'CFB','NoPadding','Ciphertext','Hex')&input=ZmY)

**Comment by n1474335** [comment]
@bwhitn I think you're right. We should try to move away from CryptoJS if we can. It's used in a number of places including the main encryption operations such as AES, DES etc, but also as the powerhouse behind our UTF8 support.

It's time we moved on to more modern libraries and web technologies, like [TextEncoder](https://developer.mozilla.org/en-US/docs/Web/API/TextEncoder) and [ArrayBuffers](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer). I'm working on a new PR which will introduce these technologies in the CyberChef core, bringing us a step closer to replacing CryptoJS entirely.

Forge looks like a good alternative for the encryption ops and we should be ok on the licence front if it can be used under BSD. I looked into Forge a while ago but decided not to use it due to its size, however now that we have the module system, this is no longer an issue.
